flink-connectors/flink-connector-filesystem/src/main/java/org/apache/flink/streaming/connectors/fs/StringWriter.java

```diff
@@ -40,12 +40,15 @@
 
 	private transient Charset charset;
 
+	private byte[] rowDelimiter ;
+
+
 	/**
 	 * Creates a new {@code StringWriter} that uses {@code "UTF-8"} charset to convert
 	 * strings to bytes.
 	 */
 	public StringWriter() {
-		this("UTF-8");
+		this("UTF-8",new byte[] {'\n'});
```

Comment:
There should exist a space before `new byte[] {'\n'}`

Reply:
good suggestion